lib/android_apk.rb

```diff
@@ -132,8 +132,16 @@ def self.analyze(filepath)
     apk.version_name = vars["package"]["versionName"]
 
     # platforms
-    apk.sdk_version = vars["sdkVersion"]
-    apk.target_sdk_version = vars["targetSdkVersion"]
+    apk.sdk_version = if vars["sdkVersion"].kind_of?(Array)
```

Comment:
A manifest has a duplicated `uses-sdk` is kinda out of specifications of Android even though it's *compile-able*! It's not well-defined which definition android framework will use, it may be potentially a flaw to easily cause a breaking change.

This gem should throw an error as well as a duplicated application tag error.

Reply:
未定義動作でこれをトラックすると大変なので、エラーを投げるようにしましょう 🙏

Reply:
thanks review! I got it.
Is the exception a `AndroidManifestValidateError`? 🐰

Reply:
Sounds good! and we have to modify ruby docs as well.

Reply:
@jmatsu 
thanks!! 👍  I done it. https://github.com/DeployGate/android_apk/pull/22/commits/55aea8d880ea7c8466c203d95ab8a6f333c5b88d
Could you review again?